projects/tinkerbell/tinkerbell-chart/chart/templates/hegel/deployment.yaml

```diff
@@ -18,7 +18,6 @@ spec:
     spec:
       containers:
         - args:
-          - --data-model=kubernetes
```

Comment:
I'm guessing the new hegel version doesn't need this anymore?

Reply:
Correct